src/views/NotificationsView/NotificationsView.js

```diff
@@ -1,5 +1,36 @@
-import React from 'react';
+import { findNotificationsForUser, findAllNotifications } from '../../services/notifications-service';
+
+import React, {useEffect, useState} from 'react';
+import Notifications from "../../components/Notifications/index.js";
+import {useSelector} from "react-redux";
+
+/**
+ * Creates a page that displays all of the notifications for a given user 
+ */
 const NotificationsView = () => {
-  return <h1>Notifications Screen</h1>;
+    const [notifications, setNotifications] = useState([]);
+    const [setError] = useState();
+
+    const authUser = useSelector((state) => state.user.data);
+
+    // find all the notifications for a given user
+    const findMyNotifications = async () => {
+        const res = await findNotificationsForUser(authUser.id);
+        if (res.error) {
+            return setError(
```

Comment:
Non-standard use of useState. Also, to be more consistent, where we return setError, we could dispatch a redux action to update the error state in the entire application, as such:
Instead of ===> const [setError] = useState()
We do ===> const dispatch = useDispatch()
Then in the return statement ====> dispatch(updateGlobalError(errorStringHere);
We'd import the methods at the top. See the file "MockInbox" for an example.

Reply:
I can try to change it, but looking at the mockInbox didn't really help, I would need you to elaborate more. We already have PRs that contain the same logic that I used, so I would prefer put this as a lower style priority until next week so I can merge this PR. Until this is PR is merged, I am blocked on some of my other tasks.